packages/lit-dev-content/src/icons/back-arrow-icon.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {svg} from 'lit';
+
+// Source: https://material.io/resources/icons/?icon=arrow_back
+export const backArrowIcon = svg`
```

Comment:
I know this is based on the other icon modules here that I wrote (sorry!), but I think these should all actually be exporting `html` tagged templates, not `svg`, right? I think `svg` is only needed for creating elements inside the svg namespace. But here since we're returning an `<svg>` element, which is HTML namespace, so we want `html`?